lib/lookbook/tags/after_render_tag.rb

```diff
@@ -0,0 +1,7 @@
+module Lookbook
+  class AfterRenderTag < YardTag
+    def value
+      text.sub(/\A:/, "")
```

Comment:
Originally, I tried going down a path that utilized `Lookbook::MethodResolver` and similar and kept running into errors and infinite loops, so I gave up and went for a simple string substitution. :P

Reply:
Ha yeah I think the simple approach looks just fine here 🙂